scripts/travis/codegen_verification.sh

```diff
@@ -42,25 +42,34 @@ function runGoFmt() {
 }
 
 function runGoLint() {
-    warningCount=$("$GOPATH"/bin/golangci-lint -c .golangci.yml | wc -l | tr -d ' ')
+    "$GOPATH"/bin/golangci-lint run -c .golangci.yml > temp-golangci-lint-results.txt
```

Comment:
It seems like it is simpler/easier to run golangci-lint and fail if it has a nonzero error code, and let the user go look in CI for the log output rather than try and capture and cat it out like this

Reply:
i agree. I'm testing that now.